code/Common/KillZoneComponent.cs

```diff
@@ -10,7 +10,7 @@ public void OnTriggerEnter( Collider other )
 		if ( other.GameObject.Components.TryGet( out Grub grub, FindMode.EverythingInSelfAndAncestors ) )
 			grub.Health.TakeDamage( GrubsDamageInfo.FromKillZone( 9999 ), true );
 
-		DestroyObjectWithTags( other.GameObject, "projectile", "drop" );
+		DestroyObjectWithTags( other.GameObject, "projectile", "drop", "landmine" );
```

Comment:
The landmine can have the projectile tag

Reply:
Fixed in next commit